src/node_internals.h

```diff
@@ -948,6 +952,26 @@ void StopProfilerIdleNotifier(const v8::FunctionCallbackInfo<v8::Value>& args);
 void Umask(const v8::FunctionCallbackInfo<v8::Value>& args);
 void Uptime(const v8::FunctionCallbackInfo<v8::Value>& args);
 
+void EnvDeleter(v8::Local<v8::Name> property,
```

Comment:
Eventually everything in node.cc that relates to the `process` object should be moved over to `node_process.cc`. Once that is done, we shouldn't need the declarations in `node_internals.h` any more and definitely won't need a `node_process.h`. These definitions are transitional only since `SetupProcessObject` is still in `node.cc`. I won't move that until after @addaleax's options parser stuff lands.